packages/app/src/backup/services/sync-backup-create.service.ts

```diff
@@ -142,20 +140,25 @@ export class SyncBackupCreateService {
   }
 
   private async waitUntilBackupDeviceFinished(
-    id: string
+    id: string,
+    iteration = 0
   ): Promise<ResultObject<GetBackupDeviceStatusResponseBody>> {
     const result = await this.getBackupDeviceStatus({
       id,
     })
 
-    if (!result.ok || result.data?.state === BackupState.Error) {
+    if (
+      !result.ok ||
+      result.data?.state === BackupState.Error ||
+      iteration > 60
```

Comment:
why 60? Does it mean something?

Reply:
I think that will be nice to provide a CONST variable to describe what it is.

Reply:
Good catch. Resolved in 42e6390e76b22a3456b1ce5b1efdbf9da9d4e2e8